src/main/webapp/app/exercises/quiz/manage/short-answer-question/short-answer-question-edit.component.ts

```diff
@@ -343,13 +350,14 @@ export class ShortAnswerQuestionEditComponent implements OnInit, OnChanges, Afte
 
     /**
      * @function addSpotAtCursor
+     * @param type Type of the spot that will be added
      * @desc Add the markdown for a spot at the current cursor location and
      * an option connected to the spot below the last visible row
      */
-    addSpotAtCursor(): void {
+    addSpotAtCursor(type: SpotType): void {
         const editor = this.questionEditor.getEditor();
         const optionText = editor.getCopyText();
-        const addedText = '[-spot ' + this.numberOfSpot + ']';
+        const addedText = `[-spot${type === SpotType.NUMBER ? '-number' : ''} ` + this.numberOfSpot + ']';
```

Comment:
Why do you not use a format string for the whole line?

Reply:
I changed only the part that I needed. But I've just updated the whole line to use string interpolation instead.